### PR TITLE
Attachment Fixes [#185195350]

### PR DESCRIPTION
**Pull request by SMaderSPIDA**

### CODE REVIEW

**Review by spidaJeffSeifert — APPROVED** [review]

**Review by SPIDABrianB — APPROVED** [review]

File: src/main/groovy/grails/plugins/mail/MailMessageBuilder.groovy [hunk 1]
```diff
@@ -385,6 +385,8 @@ class MailMessageBuilder {
     }
 
     protected doAdd(String id, String contentType, InputStreamSource toAdd, boolean isAttachment) {
+        mailSender = grails.util.Holders.grailsApplication.mainContext.getBean("mailSender")
```
Comment (CharlieASpida) [inline src/main/groovy/grails/plugins/mail/MailMessageBuilder.groovy:388]: why is this needed? mailSender should be autowired in the MailGrailsPlugin
